src/output_formatters.py

```
@@ -0,0 +1,109 @@
+
+    Args:
+        pattern (str): The search pattern used.
+        flags (dict): Dictionary of CLI flags and options.
+
+    Methods:
+        format(results): Convert match results to a JSON string.
+    """
+
+    # pylint: disable=too-few-public-methods
+
+    def __init__(self, pattern: str, flags: dict):
```

Ruff (0.14.1)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)